docs/src/release-notes-js.md

```diff
@@ -6,6 +6,72 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.49
+
+### New Chromium Headless
+
+Prior to this release, Playwright was running the old established implementation of Chromium headless. However, Chromium had entirely switched to the [new headless mode](https://developer.chrome.com/docs/chromium/headless) implementation, so Playwright had to switch as well.
+
+Most likely, this change should go unnoticed for you. However, the new headless implementation differs in a number of ways, for example when handling pdf documents, so please file an issue if you encounter a problem.
+
+### Chromium Headless Shell
+
+Playwright now also ships `chromium-headless-shell` channel that is a separate build that closely follows the old headless implementation. If you would like to keep the old behavior before you are ready to switch to the new headless, please fallback to this channel:
+
+1. First, install this channel prior to running tests. Make sure to list all browsers that you use.
+
+  ```bash js
+  # running tests in all three browsers, headless and headed
+  npx playwright install chromium firefox webkit chromium-headless-shell
+  ```
+
+  If you are not running headed tests on CI, you can omit `chromium` from the list. Similarly, you can omit other unused browsers.
```

Comment:
I think we should reverse since the majority of folks doesn't run headed tests on CI -> we can remove `chromium` from the list.

Reply:
But the line above is not CI-specific, that's what you have to do locally as well.